package.json

```diff
@@ -126,13 +126,13 @@
   },
   "devDependencies": {
     "@clack/prompts": "^0.7.0",
-    "@mdit-vue/plugin-component": "2.0.0",
-    "@mdit-vue/plugin-frontmatter": "2.0.0",
-    "@mdit-vue/plugin-headers": "2.0.0",
-    "@mdit-vue/plugin-sfc": "2.0.0",
-    "@mdit-vue/plugin-title": "2.0.0",
-    "@mdit-vue/plugin-toc": "2.0.0",
-    "@mdit-vue/shared": "2.0.0",
+    "@mdit-vue/plugin-component": "2.1.0",
```

Comment:
feel free to use semver ranges, `^2.1.0` . I had locked because it was causing some issues 👀

Reply:
Updated